src/lib/constant_mappings.hpp

```diff
@@ -23,6 +24,8 @@ extern const std::unordered_map<OrderByMode, std::string> order_by_mode_to_strin
 extern const std::unordered_map<hsql::OrderType, OrderByMode> order_type_to_order_by_mode;
```

Comment:
Just a general question: you're using a lot of unordered maps. Are they really for all the cases the best fit over a `std::map`? Especially for growing a insert-heavy workloads, the might be better off with a map (I have not idea what you're doing here, so it's just a general question out of curiosity).

Reply:
This is just a mapping of constants because C++ doesn't have a good way to convert enums to strings and back.

Reply:
Ok.